components/periodic-table/project/periodic_table/PeriodicTable.vue

```diff
@@ -173,6 +215,86 @@ export default {
         this.periodicTableMQL.addListener(handler) // for Safari 14 and earlier
       }
     },
+    /**
+     * カーソルがホバーしているセルにフォーカスする
+     */
+    handleElementHover(atomicNumber) {
+      this.focusedAtomicNumber = atomicNumber;
+    },
+
+    /**
+     * ↑ ↓ で同族の前・次の元素に行く
+     */
+    findNextInGroup(direction) {
+      const currentElement = this.elementList.find(e => e.atomicNumber === this.focusedAtomicNumber);
+      if (!currentElement) return this.focusedAtomicNumber;
+
+      const groupElements = this.groupNavigation[currentElement.group];
+      if (!groupElements) return this.focusedAtomicNumber;
+
+      const currentIndex = groupElements.indexOf(this.focusedAtomicNumber);
+      if (currentIndex === -1) return this.focusedAtomicNumber;
+
+      const nextIndex = direction === 'up' ? currentIndex - 1 : currentIndex + 1;
+
+      if (nextIndex >= 0 && nextIndex < groupElements.length) {
+        return groupElements[nextIndex];
+      }
+
+      return this.focusedAtomicNumber;
+    },
+
+    /**
+     * キーボード操作
+     * 上下左右で移動、スペースキー・リターンキーで詳細を開ける
+     */
+    handleKeyDown(event) {
+      const key = event.key;
+      let nextAtomicNumber = this.focusedAtomicNumber;
+
+      switch (key) {
+        case 'ArrowRight':
+          if (nextAtomicNumber < 118) nextAtomicNumber++;
+          event.preventDefault();
+          break;
+        case 'ArrowLeft':
+          if (nextAtomicNumber > 1) nextAtomicNumber--;
+          event.preventDefault();
+          break;
+        case 'ArrowUp': {
+          // ランタノイドやアクチノイドの場合は ↑ でも ← でも前の元素（原子番号-1）に行きます
```

Comment:
nits
```suggestion
          // ランタノイドやアクチノイドの場合は ↑ でも ← でも前の元素（原子番号-1）に行く
```

下のコメントも同様です